### ISSUE TITLE
Determine the date/time for countdown

### ISSUE THREAD

**Issue by aulneau** [blockstack/app.co#321]
![screen shot 2018-12-06 at 9 22 28 am](https://user-images.githubusercontent.com/11803153/49593144-7c147380-f938-11e8-87c7-374b3e5303c1.png)

What date and time should this count down to? additionally, can we set up a schedule/calendar that the API can return? cc @jeffdomke @GinaAbrams @hstove

**Comment by GinaAbrams** [comment]
Like that option as a current go-to @jeffdomke 👍 

Possible to do?

**Comment by jeffdomke** [comment]
In that case can you please furnish all the ~final dates for 2019?

**Comment by GinaAbrams** [comment]
Here in the [doc](https://paper.dropbox.com/doc/App-Mining-Roadmap--ATMPzjx1WG7~9yxy3UmuJzK6Ag-p2P2fBStxpnV57TYE1QAK)

**Comment by jeffdomke** [comment]
@aulneau Now I see this working I'm finding the time labels a bit visually hard to parse. Two suggestions: 

- lowercase "D" "H" "M" "S" 
– change color for those labels. The blue or 60%. Somehow reduced visually.

**Comment by aulneau** [comment]
I tried both, I think lower case is not my favorite:

<img width="571" alt="screen shot 2018-12-11 at 5 06 44 pm" src="https://user-images.githubusercontent.com/11803153/49833525-c1390b00-fd67-11e8-84e2-974f58db8d54.png">

<img width="607" alt="screen shot 2018-12-11 at 5 10 38 pm" src="https://user-images.githubusercontent.com/11803153/49833527-c1390b00-fd67-11e8-9506-9e7ead598861.png">

What do you think?

**Comment by GinaAbrams** [comment]
@aulneau possible for us to edit this ticker to reflect end of day today as the end of the registration period? Also, I only see it when I scroll through very specific parts of the page, not all the time, and would like for it to be a permanent bar at the top if possible. Thoughts on this? cc @jeffdomke

**Comment by jeffdomke** [comment]
Yes to both please.

**Comment by aulneau** [comment]
Yep I'll do so this morning. The counter is attached to the header which will be present when the user scrolls up. I can just make it fixed in all states.

**Comment by GinaAbrams** [comment]
Thanks @aulneau 🙏

**Comment by aulneau** [comment]
@GinaAbrams will be resolved with https://github.com/blockstack/app.co/pull/389
